python/pyarrow/tests/test_table.py

```diff
@@ -2204,3 +2204,100 @@ def test_table_cast_invalid():
 
     table = pa.table({'a': [None, 1], 'b': [False, True]})
     assert table.cast(new_schema).schema == new_schema
+
+
+def test_table_sort():
```

Comment:
There is already a `test_table_sort_by` a little bit above. Can you move those new tests over there?

(this Table test might also be redundant with what there already is)